res/values/strings.xml

```diff
@@ -520,6 +520,12 @@
     <string name="media_service_notification_channel_description">See music player</string>
     <string name="file_sync_notification_channel_name">File sync</string>
     <string name="file_sync_notification_channel_description">See file sync result</string>
+
+    <string name="rate_dialog_title">Rate ownCloud app!</string>
```

Comment:
@harshitbansal05 ownCloud should not appear as name, in case of branding (thinking a bit in the future, so the option will be disabled). It should be used the `app_name`parameter.